src/routes/products/allIds.js

```diff
@@ -0,0 +1,48 @@
+const URL = require("url");
+const querystring = require("querystring");
+
+const fs = require("fs");
+const path = require("path");
+
+const idProductRoute = (req, res) => {
+  const productsPath = path.join(
+    __dirname,
+    "../../",
+    "db",
+    "products",
+    "all-products.json"
+  );
+
+  fs.readFile(productsPath, "utf8", (err, data) => {
+    try {
+      const allProducts = JSON.parse(data);
+      const reqUrl = URL.parse(req.url);
+      const queryStr = querystring.parse(reqUrl.query);
+      const queryIds = queryStr.ids
+        .slice(1, queryStr.ids.length - 1)
+        .split(",");

```

Comment:
didn't find a way parse URL with quotes without wrapper into the one more quotes